SparkPost/2web2ui
# EO-800 Create the new Alerts List page (only use old API)

### What Changed
 - Created the new Alerts List page under `/alerts-new`.
 - Edited the current table to match the styling of the new table designs. 
     - This is done by adding a custom render function that renders components differently if given. 
 - ~Fixed the alerts toggle to use the redux store as the single source of truth.~ Will do in another bug ticket
 - Added a user feature flag for the new alerts page

### How To Test
 - Enable user feature flag for the account (or comment out line 714 in `src/config/routes`.
 - navigate to `/alerts-new`. 
 - Verify that the app matches the design.

### Notes
 - This is for subtask EO-800. This only implements design features. The actual data from the API will be in another PR. I have left TODOs in places where changes will need to be made once the API is deployed. I also have not written tests that require API data to be mocked.  

### To Do
- [x] Look into using separate methods for opening/closing delete modal 
- [x] Fix routes merge conflict
